public/app/features/templating/state/uuidInEditorReducer.ts

```diff
@@ -0,0 +1,19 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { VariablePayload } from './actions';
+
+const uuidInEditorReducerSlice = createSlice({
+  name: 'templating/uuidInEditor',
+  initialState: null,
+  reducers: {
+    changeToEditorEditMode: (state, action: PayloadAction<VariablePayload>) => {
+      return action.payload.uuid;
+    },
+    changeToEditorListMode: (state, action: PayloadAction<undefined>) => {
+      return null;
+    },
+  },
+});
```

Comment:
Out of curiosity as I see quite some UI state in the Redux - any particular reasons to store it in Redux? Or just convention to keep everything templating related in store?

Reply:
The thought is that it get more tests that way so we try to keep the component super stupid. This particular reducer is no more.